internal/identity/oidc/onelogin/onelogin.go

```diff
@@ -12,62 +13,66 @@ import (
 	"golang.org/x/oauth2"
 
 	"github.com/pomerium/pomerium/internal/httputil"
+	"github.com/pomerium/pomerium/internal/identity"
+	pom_oidc "github.com/pomerium/pomerium/internal/identity/oidc"
 	"github.com/pomerium/pomerium/internal/sessions"
 	"github.com/pomerium/pomerium/internal/version"
 )
 
+var _ identity.Authenticator = &OneLoginProvider{}
+
+// Name identifies the OneLogin identity provider
+const Name = "onelogin"
+
 const defaultOneLoginProviderURL = "https://openid-connect.onelogin.com/oidc"
 const defaultOneloginGroupURL = "https://openid-connect.onelogin.com/oidc/me"
```

Comment:
@desimone very nice work so far! we can implement this suggestion here too https://github.com/pomerium/pomerium/pull/560/files#r398815591